ApollosProject/apollos-prototype
# Filters by active in content items data source

## DESCRIPTION

### What does this PR do, or why is it needed?
In the past, items would show up in the app before they went live or after they expire. Many churches use those dates to force content to go live or to remove content from their feed, so we need to respect the values of those properties. This PR does that. 

I also added some methods to the `request-builder.js` so that request filter chaining should be more concise and read better.

### What design trade-offs/decisions were made?
n/a

### How do I test this PR?
1. Open the app w/ local API.
2. Examine the content item feed and pick an item.
3. In Rock, edit the Active At or Expires At dates to make the item inactive.
4. Refresh the content feed in the app, the item should disappear.  

### What automated tests did you write?

Updated existing tests.

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
